_episodes/08-putting-it-all-together.md

```diff
@@ -179,32 +181,45 @@ or create matplotlib `figure` and `axis` objects first and add the plot later on
 
 ~~~
 fig, ax = plt.subplots()  # initiate an empty figure and axis matplotlib object
-ax.plot(x, y, '-')
+ax.hist(s, 30)
```

Comment:
```suggestion
ax.hist(s, 30) # 30 is the number of bins, or discrete categories, into which X-axis is broken
```

Reply:
I think putting it in a standard text would be better. Something like
> where `30` is the number of bins in the histogram.